deps/libcloop/src/cserver.c

```diff
@@ -613,8 +613,8 @@ cconn_on_write(cloop_ent_t *clent, int ev)
 			continue;
 		}
 		if (cserver_debug) {
-			fprintf(stderr, "CCONN[%p] WRITE DATA: HAVE %d; "
-			    "Q %d\n", ccn, want,
+			fprintf(stderr, "CCONN[%p] WRITE DATA: HAVE %ld; "
```

Comment:
Here and generally, including libcbuf changes: these should be %zu
as size_t is unsigned, and that way we won't break 32-bit compiles.

We might also want to consider adding -m64 to the Makefiles so we're
explicit about what mode we're compiling in.